examples/v0/account_management/create_customer.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example illustrates how to create a new customer under a given
+manager account.
+
+Note: this example must be run using the credentials of a Google Ads manager
+account. By default, the new account will only be accessible via the manager
+account.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+from datetime import datetime
+
+import google.ads.google_ads.client
+
+
+def main(client, manager_customer_id):
+    customer_service = client.get_service('CustomerService')
+    customer = client.get_type('Customer')
+    today = datetime.today().strftime('%Y%m%d %H:%M:%S')
+    customer.descriptive_name.value = ('Account created with '
+                                       'CustomerService on %s' % today)
+    # For a list of valid currency codes and time zones see this documentation:
+    # https://developers.google.com/adwords/api/docs/appendix/codes-formats
+    customer.currency_code.value = 'USD'
+    customer.time_zone.value = 'America/New_York'
+    # The below values are optional. For more information about URL
+    # options see: https://support.google.com/google-ads/answer/6305348
+    customer.tracking_url_template.value = '{lpurl}?device={device}'
+    customer.final_url_suffix.value = ('keyword={keyword}&matchtype={matchtype}'
+                                       '&adgroupid={adgroupid}')
+    customer.has_partners_badge.value = False
+
+    try:
+        response = customer_service.create_customer_client(
+            manager_customer_id, customer)
+        print(('Customer created with resource name "%s" under manager account '
+               'with customer ID %s') %
```

Comment:
**nitpick**: Resource names and IDs are usually wrapped in quotations.

Reply:
Ah so like `'with customer ID "%s"'`? I'll update here, and in the other PR too if needed.